src/platform/silabs/platformAbstraction/WiseMcuSpam.cpp

```diff
@@ -36,8 +35,13 @@ extern "C" {
 #include "em_core.h"
 #include "rsi_board.h"
 #include "sl_event_handler.h"
+
+#ifdef SL_CATALOG_SIMPLE_BUTTON_PRESENT
 #include "sl_si91x_button.h"
 #include "sl_si91x_button_pin_config.h"
+#endif  //SL_CATALOG_SIMPLE_BUTTON_PRESENT
+
+#ifdef ENABLE_WSTK_LEDS
```

Comment:
Why not combine this in the #if just 1line under

Reply:
Added in 1 line